docs/how-tos/dbt_extensions.rst

```diff
@@ -1,14 +1,47 @@
 .. _dbt-extensions:
 
-DBT extensions
-**************
-
-To extend the DBT project, you can use the following Tutor variables:
-
-- **DBT_REPOSITORY**: A git repository URL to clone and use as the DBT project.
-- **DBT_BRANCH**: The branch to use when cloning the DBT project.
-- **DBT_PROJECT_DIR**: The directory to use as the DBT project.
-- **EXTRA_DBT_PACKAGES**: A list of python packages for the DBT project to install.
-- **DBT_ENABLE_OVERRIDE**: This variable determines whether the DBT project override feature
-  should be enabled or not. When enabled, it allows you to make changes to the **dbt_project.yml**
-  and **packages.yml** files using the tutor patches: `dbt-packages` and `dbt-project`.
+Extending dbt
+*************
+
+As noted in :ref:`dbt`, you can install your own custom ``dbt`` package to apply your own transforms to the event data
+in Aspects.
+
+**Step 1. Create your dbt package**
+
+See `Building dbt packages`_ for details.
+
+**Step 2. Link to aspects-dbt**
+
+Aspects charts depend on the transforms in `aspects-dbt`_, so it's important that your ``dbt`` package also installs
```

Comment:
```suggestion
Aspects charts depend on the transforms in `aspects-dbt`_, so it's important that your ``dbt`` package also installs the same version of `aspects-dbt`_ as your `tutor_contrib_aspects` plugin uses.
```

Reply:
Good point -- I updated "step 2" to better illustrate what those `package.yml` values should be:

https://docsopenedxorg--111.org.readthedocs.build/projects/openedx-aspects/en/111/how-tos/dbt_extensions.html